Running `antora generate site-workspace.yml` fails

I get:
```
error: Cannot find module 'asciidoctor-plantuml'
Add the --stacktrace option to see the cause.
```

According to the docs, I started a PlantUML server as a docker container, which is running:
```
→ docker ps
CONTAINER ID        IMAGE                            COMMAND                  CREATED              STATUS              PORTS                    NAMES
892defe58534        plantuml/plantuml-server:jetty   "/docker-entrypoint.…"   About a minute ago   Up About a minute   0.0.0.0:9080->8080/tcp   cool_raman
```

What am I missing?

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "asciidoctor-plantuml": "^1.1.13",
+    "asciidoctor-plantuml": "^1.5.0",
     "asciidoctor.js": "^1.5.9"
   }
 }

